### DISCUSSION TITLE
Strategy for transitioning existing project to tailwind

### DISCUSSION THREAD

**Discussion by axelclark** [tailwindlabs/tailwindcss#1857] [Help]
I have an existing project created with another CSS Framework.  Does anyone have any strategies for transitioning to tailwind over time?

**Comment by estevanmaito** [comment]
If you're coming from Bootstrap specifically, there's this project that converts BS classes to Tailwind https://github.com/awssat/tailwindo

**Reply by axelclark** [reply]
My project uses [Bourbon](https://www.bourbon.io/) and [Bitters](https://github.com/thoughtbot/bitters), but the project you listed looks great for those coming from Bootstrap!

**Comment by mlieshout** [comment]
I've used Shift Tailwind Converter to convert an existing BS4 site to Tailwind:

https://laravelshift.com/convert-bootstrap-to-tailwind-css

Works really well

**Comment by MatthewMcPCyNation** [comment]
Did you ever get a strategy for this? We're hoping to do the same (from bourbon/bitters). 

Currently we're thinking of sandboxing our css and applying tailwind to each specific component, removing bourbon as we go.

**Reply by axelclark** [reply]
I never found a great strategy to do it over time.  I ended up doing it in a bunch of small commits, but one big pull request.  If you have a strategy for sandboxing your css, I think you should be good updating by component.
